Source/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -226,6 +238,16 @@ static void CollectAccessibilityElements(ASDisplayNode *node, NSMutableArray *el
   }
 
   for (ASDisplayNode *subnode in node.subnodes) {
+    // If a node is hidden or has an alpha of 0.0 we should not include it
+    if (subnode.hidden || subnode.alpha == 0.0) {
+        continue;
+    }
+    // If a subnode is outside of the view's window, exclude it
+    CGRect nodeInWindowCoords = [node convertRect:subnode.frame toNode:nil];
+    if (view.window && !CGRectIntersectsRect(view.window.frame, nodeInWindowCoords)) {
```

Comment:
Does it make sense to exit early if the view has no window? By requiring a window before checking to see if the view is in its bounds I'm keeping the behavior the same as before, but not sure if it is correct.

Reply:
I think it makes sense to exit early.

Reply:
Updated! I had to update a few accessibility tests to make sure the node we are testing was in a window.